provisioning/service/samples/getting started/EnrollmentGroupSample/EnrollmentGroupSample.cs

```diff
@@ -3,87 +3,88 @@
 
 using System;
 using System.Linq;
-using System.Security.Cryptography.X509Certificates;
 using System.Threading.Tasks;
+using Newtonsoft.Json;
 
 namespace Microsoft.Azure.Devices.Provisioning.Service.Samples
 {
     internal class EnrollmentGroupSample
     {
-        private const string EnrollmentGroupId = "enrollmentgrouptest";
+        private static readonly string s_enrollmentGroupId = $"EnrollmentGroupSample-{Guid.NewGuid()}";
         private readonly ProvisioningServiceClient _provisioningServiceClient;
-        private readonly X509Certificate2 _groupIssuerCertificate;
 
-        public EnrollmentGroupSample(ProvisioningServiceClient provisioningServiceClient, X509Certificate2 groupIssuerCertificate)
+        public EnrollmentGroupSample(ProvisioningServiceClient provisioningServiceClient)
         {
             _provisioningServiceClient = provisioningServiceClient;
-            _groupIssuerCertificate = groupIssuerCertificate;
         }
 
         public async Task RunSampleAsync()
         {
-            await QueryEnrollmentGroupAsync().ConfigureAwait(false);
+            await CreateEnrollmentGroupAsync();
```

Comment:
Create the enrollment before querying, to ensure there is at least one to query.

Don't use ConfigureAwait() in samples.